packages/block-library/src/editor.scss

```diff
@@ -58,13 +58,6 @@
  * This allows us to create normalization styles that are easily overridden by editor styles.
  */
 
-// Provide every block with a default base margin. This margin provides a consistent spacing
-// between blocks in the editor.
-.editor-styles-wrapper .block-editor-block-list__block {
-	margin-top: $default-block-margin;
-	margin-bottom: $default-block-margin;
```

Comment:
I've been wanting to remove this for ages, but there's always been some reason for keeping it. Can it safely be removed now? Cc @jasmussen

Reply:
It seeems like we can! From quick testing in a theme that doesn't style the editor, things look right. And it'd definitely a rule we want to remove. So it does seem now's the time!

Note that these two rules seem to mostly take its place:

<img width="203" alt="Screenshot 2020-03-09 at 13 55 09" src="https://user-images.githubusercontent.com/1204802/76214980-10414a80-620e-11ea-9780-f9970b4f39e4.png">

and

<img width="227" alt="Screenshot 2020-03-09 at 13 57 06" src="https://user-images.githubusercontent.com/1204802/76214987-13d4d180-620e-11ea-8d06-4b68769da41e.png">

Good to be mindful of these removals, when we get to the DOM "lightening", can check on a per-block basis, because now it will be up to each individual block to ensure a margin.